src/components/BugsTable/index.jsx

```diff
@@ -0,0 +1,270 @@
+          'JS',
+          'None',
+          '2018-02-12'
+        ),
+        createData(
+          'Servo',
+          '1441960 - Add measure of time to start processing a task',
+          'JS',
+          'None',
+          '2018-05-06'
+        ),
+        createData(
+          'Servo',
+          '1446768 - Only post "No taskcluster jobs.." to a PR once',
+          'JS',
+          'None',
+          '2018-05-01'
+        ),
+        createData(
+          'Taskcluster',
+          '1441977 - Run tests for taskcluster-treeherder in Taskcluster',
+          'JS',
+          'None',
+          '2018-05-03'
+        ),
+      ].sort((a, b) => (a.lastupdate > b.lastupdate ? -1 : 1)),
+      page: 0,
+      rowsPerPage: 5,
+    };
+  }
+
+  handleRequestSort = (event, property) => {
+    const orderBy = property;
+    let order = 'desc';
+
+    if (this.state.orderBy === property && this.state.order === 'desc') {
+      order = 'asc';
+    }
+
+    const data =
+      order === 'desc'
+        ? this.state.data.sort((a, b) => (b[orderBy] < a[orderBy] ? -1 : 1))
+        : this.state.data.sort((a, b) => (a[orderBy] < b[orderBy] ? -1 : 1));
+
+    this.setState({ data, order, orderBy });
+  };
+
+  handlePageChange = (event, page) => {
+    this.setState({ page });
+  };
+
+  handleRowsPerPageChange = event => {
+    this.setState({ rowsPerPage: event.target.value });
+  };
+
+  render() {
+    const { classes } = this.props;
+    const { data, order, orderBy, rowsPerPage, page } = this.state;
+    const emptyRows =
+      rowsPerPage - Math.min(rowsPerPage, data.length - page * rowsPerPage);
+
+    return (
+      <Paper className={classes.root}>
+        <BugsTableToolbar />
+        <div className={classes.tableWrapper}>
+          <Table className={classes.table} aria-labelledby="tableTitle">
+            <BugsTableHead
+              order={order}
+              orderBy={orderBy}
+              onRequestSort={this.handleRequestSort}
+            />
+            <TableBody>
+              {data
+                .slice(page * rowsPerPage, page * rowsPerPage + rowsPerPage)
+                .map(n => (
+                  <TableRow hover tabIndex={-1} key={n.id}>
+                    <TableCell component="th" scope="row">
+                      {n.project}
+                    </TableCell>
+                    <TableCell>{n.description}</TableCell>
+                    <TableCell>{n.tag}</TableCell>
+                    <TableCell>{n.assignedto}</TableCell>
+                    <TableCell>{n.lastupdate}</TableCell>
+                  </TableRow>
+                ))}
+              {emptyRows > 0 && (
+                <TableRow style={{ height: 49 * emptyRows }}>
+                  <TableCell colSpan={6} />
+                </TableRow>
+              )}
+            </TableBody>
+          </Table>
+        </div>
+        <TablePagination
+          component="div"
+          count={data.length}
+          rowsPerPage={rowsPerPage}
```

Comment:
Create `constants.js` in `src/utils`.

```js
export const PROJECTS_PAGE_SIZE = 25;
```

Then in ` src/components/BugsTable/index.jsx` you can do:

```js
import { PROJECTS_PAGE_SIZE } from '../../utils/constants';
...

<TablePagination
  component="div"
  count={data.length}
  rowsPerPage={PROJECTS_PAGE_SIZE}
  rowsPerPageOptions={[PROJECTS_PAGE_SIZE]}
  page={page}
  backIconButtonProps={{
    'aria-label': 'Previous Page',
  }}
  nextIconButtonProps={{
    'aria-label': 'Next Page',
  }}
  onChangePage={this.handlePageChange}
  onChangeRowsPerPage={this.handleRowsPerPageChange}
/>
```

I added the extra prop `rowsPerPageOptions={[PROJECTS_PAGE_SIZE]}` to remove the option to the user of being able to select a number of rows per page.